### PR TITLE
Set Rotate from propPanel default schema

### PR DESCRIPTION
**Pull request by hand-dot**
Instead of mandating the implementation of 'rotate' for all elements, which would increase the developer's workload, allow for the omission of 'rotate' implementation on a case-by-case basis.

If 'rotate' is set [here](https://github.com/pdfme/pdfme/blob/8e4ccc3c35c6008e5cf5ec885c90799d68a70f7c/packages/schemas/src/text/propPanel.ts#L137), enable rotate in the Designer, and if 'rotate' is not set here, make rotate unavailable.

### CODE REVIEW

File: packages/ui/src/components/Designer/Main/index.tsx [hunk 1]
```diff
@@ -242,6 +242,10 @@ const Main = (props: Props, ref: Ref<HTMLDivElement>) => {
     setEditing(true);
   };
 
+  const rotatable = schemasList[pageCursor]
```
Comment (hand-dot) [inline packages/ui/src/components/Designer/Main/index.tsx:245]: detect rotatable or not

File: packages/schemas/src/text/propPanel.ts [hunk 2]
```diff
@@ -135,6 +135,9 @@ export const propPanel: PropPanel<TextSchema> = {
     position: { x: 0, y: 0 },
     width: 45,
     height: 10,
+    // If the value of "rotate" is set to undefined or not set at all, rotation will be disabled in the UI.
+    // Check this document: https://pdfme.com//docs/custom-schemas#learning-how-to-create-from-pdfmeschemas-code
+    rotate: 0,
```
Comment (hand-dot) [inline packages/schemas/src/text/propPanel.ts:140]: set rotate as number

File: packages/ui/src/components/Designer/Sidebar/index.tsx [hunk 3]
```diff
@@ -1,33 +1,18 @@
...
   const i18n = useContext(I18nContext);
-  const font = useContext(FontContext);
-  const fallbackFont = getFallbackFontName(font);
   const [open, setOpen] = useState(true);
 
-  const getActiveSchemas = () => {
-    const ids = activeElements.map((ae) => ae.id);
-    const activeSchema = schemas.find((s) => ids.includes(s.id));
-
-    if (activeSchema?.type === 'text') {
```
Comment (hand-dot) [inline packages/ui/src/components/Designer/Sidebar/index.tsx:22]: refactoring

**Comment by peteward** [comment]
nice!

**Pull request merged**